src/node/utils/ImportHtml.js

```diff
@@ -103,6 +102,8 @@ exports.setPadHTML = function(pad, html)
   var theChangeset = builder.toString();
 
   apiLogger.debug('The changeset: ' + theChangeset);
-  pad.setText("\n");
-  pad.appendRevision(theChangeset);
+  await Promise.all([
+    pad.setText('\n'),
```

Comment:
Is it possible that this can lead to a race when `pad.appendRevision` completes before `pad.setText`?

Reply:
Looking at the code I don't think there's currently a race condition, but that could easily change if the implementation of `Pad.setText()` or `Pad.appendRevision()` is changed. Either way, this code is not very readable; waiting for `setText()` to complete before calling `appendRevision()` would be an improvement.

(Note that this code was problematic before—this PR didn't introduce the problem.)